src/components/cc-block-detail/cc-block-detail.stories.js

```diff
@@ -0,0 +1,56 @@
+import { makeStory } from '../../stories/lib/make-story.js';
+import './cc-block-detail.js';
+import '../cc-block/cc-block.js';
+
+export default {
+  tags: ['autodocs'],
+  title: '🧬 Molecules/<cc-block-detail>',
+  component: 'cc-block-detail',
+};
+
+const conf = {
+  component: 'cc-block',
+};
+
+const ccBlockHtmlExemple = `
+  <div slot="header-title">This is my block</div>
+  <div slot="content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque feugiat dui at leo porta dignissim. Etiam ut purus ultrices, pulvinar tellus quis, cursus massa. Mauris dignissim accumsan ex, at vestibulum lectus fermentum id. Quisque nec magna arcu. Quisque in metus sed erat sodales euismod eget id purus. Sed sagittis rhoncus mauris. Ut sit amet urna ac nunc semper porta. Nam ut felis eu velit luctus rutrum. Nam leo nisl, molestie a varius non, ullamcorper sit amet tortor. Donec in convallis ex. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Praesent hendrerit venenatis erat, eu malesuada nulla viverra eu. Curabitur porta risus augue, non rutrum lectus hendrerit a.</div>
+`;
+
+const defaultHtmlExample = `
+  ${ccBlockHtmlExemple}
+  <div slot="footer" style="background-color: var(--cc-color-bg-neutral); border-top: solid 2px #cdcdcd">
+    <cc-block-detail>
+      <div slot="tab-title">Command line</div>
+      <a slot="link" href="">See documentation</a>
+      <div slot="content">Praesent tincidunt dapibus elit, sed posuere risus sodales quis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Integer ultrices maximus ligula eu efficitur. Nulla luctus ipsum vitae dui gravida, sodales ornare dolor viverra. Vivamus pretium scelerisque ultricies. Nulla consectetur euismod euismod. Interdum et malesuada fames ac ante ipsum primis in faucibus. Cras dignissim id sapien non facilisis. Suspendisse vitae pretium orci. Nunc aliquam pellentesque nunc, at tincidunt nisi lobortis a. </div>
+    </cc-block-detail>
+  </div>
```

Comment:
suggestions:

* if we use the `slot="footer-left"` instead of `footer`, we'll get the gray background, the border and padding
  * we can remove the shadow in the cc-block in another commit if Marion agrees
* I we do this, we can remove the extra div and set the `slot` directly on the `cc-block-detail`

(same for the other story)